lib/matplotlib/tests/test_pyplot.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+import pytest
+import six
+import matplotlib.pyplot as plt
+from matplotlib.testing.decorators import image_comparison
+
+
+@image_comparison(baseline_images=["explict_axes"],
+                  style='mpl20', extensions=['svg'],
+                  remove_text=True)
+def test_explict_ax():
+    fig, (ax1, ax2) = plt.subplots(2)
+    plt.plot(range(5))
+    plt.plot(range(5)[::-1], ax=ax1)
```

Comment:
I would just test that len(ax1.lines) == 2 instead of doing an image comp.

test name has typo ("explicit")

Reply:
There should be one line per axes (`gca()` is surprising! ax2 was created last so it is 'current')

Reply:
you got me :-)
but still you should be able to just test how many lines there are on each axis.

Reply:
Done